pytorch-notebook/environment.yml

```diff
@@ -7,6 +7,6 @@ channels:
 dependencies:
  - jupyterlab-nvdashboard
  - gpytorch
- - pytorch>=2.0.0=*cuda112*
- - torchvision>=0.15.1=*cuda112*
+ - pytorch>=2.0.0=*cuda118*
```

Comment:
Apparently we've been on 11.8 for some time, so perhaps this * syntax isn't really pinning anything? https://github.com/pangeo-data/pangeo-docker-images/blob/d2651fffe3f564eda66f873c8d052254b9f6d6de/pytorch-notebook/packages.txt#L114-L115

Reply:
That's cudatoolkit. CUDA 11.2 and 11.8 does have this forward/backward compatibility thing, but if you look at the conda-lock.yml file, the Pytorch build is actually the CUDA 11.2 one:

https://github.com/pangeo-data/pangeo-docker-images/blob/d2651fffe3f564eda66f873c8d052254b9f6d6de/pytorch-notebook/conda-lock.yml#L9696